.github/workflows/check-migrations.yml

```diff
@@ -5,14 +5,18 @@ on:
     types:
       - labeled
   workflow_dispatch:
+# Set default permissions as restrictive
+permissions:
```

Comment:
GitHub now suggests having permissions on every workflow, or considers it a potential security risk